lib/pages/portal/service/website_provider.dart

```diff
@@ -103,6 +104,52 @@ class WebsiteProvider with ChangeNotifier {
     }
   }
 
+
+  /// Initializes the number of visits of websites with the value stored locally.
+  /// Because [PrefService] doesn't support storing maps, the
+  /// data is stored in 2 lists: the list of website IDs ([websiteIds]) and the list
+  /// with the number of visits ([websiteVisits]), where `websiteVisits[i]` is the
+  /// number of times the user accessed website with ID `websiteIds[i]`.
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph
```suggestion
  /// Initializes the number of visits of websites with the value stored locally.
  ///
  /// Because [PrefService] doesn't support storing maps, the
  /// data is stored in 2 lists: the list of website IDs ([websiteIds]) and the list
  /// with the number of visits ([websiteVisits]), where `websiteVisits[i]` is the
  /// number of times the user accessed website with ID `websiteIds[i]`.
```